### PR TITLE
chore: make pod and package.json versions more deterministic

### PR DESCRIPTION
**Pull request by bryce-mcmath**
Clecio and I found that having ranges on npm package versions in `package.json` was causing the `pod install` to inconsistently generate pod versions in `Podfile.lock`. I've removed the carets from our npm versions and updated relevant versions so that no podfile versions changed

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -3705,19 +4216,6 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@hyperledger/indy-vdr-react-native@patch:@hyperledger/indy-vdr-react-native@npm:0.1.0#./.yarn/patches/@hyperledger-indy-vdr-react-native-npm-0.1.0-10c696b320.patch::locator=bc-wallet-mobile%40workspace%3A.":
```
Comment (wadeking98) [inline yarn.lock]: seems like the patch is deleted here

File: app/package.json [hunk 2]
```diff
@@ -58,109 +58,109 @@
...
-    "@hyperledger/aries-oca": "1.0.0-alpha.150",
-    "@hyperledger/aries-react-native-attestation": "1.0.0-alpha.150",
-    "@hyperledger/indy-vdr-react-native": "^0.1.0",
+    "@hyperledger/anoncreds-react-native": "0.1.0",
+    "@hyperledger/aries-askar-react-native": "0.1.1",
+    "@hyperledger/aries-bifold-core": "1.0.0-alpha.155",
+    "@hyperledger/aries-bifold-verifier": "1.0.0-alpha.155",
+    "@hyperledger/aries-oca": "1.0.0-alpha.155",
+    "@hyperledger/aries-react-native-attestation": "1.0.0-alpha.155",
+    "@hyperledger/indy-vdr-react-native": "0.1.0",
```
Comment (wadeking98) [inline app/package.json:67]: maybe changing indy-vdr-react-native back to `^0.1.0` would fix the issue?

**Comment by wadeking98** [comment]
@bryce-mcmath The error you're seeing during initialization is because the patch isn't being applied to `@hyperledger/indy-vdr-react-native`

File: app/package.json (hunk 2 above)
Comment (bryce-mcmath) [reply app/package.json:67]: instead I removed the caret from the resolution block in the top level package.json and that worked

File: yarn.lock (hunk 1 above)
Comment (bryce-mcmath) [reply yarn.lock]: it's been added back now after I removed the caret from the resolution block

**Review by wadeking98 — APPROVED** [review]

**Pull request merged**